q2_feature_table/_examples.py

```diff
@@ -0,0 +1,278 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2022, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+from biom import Table
+
+import qiime2
+from qiime2 import Artifact
+
+
+rep_seqs_1_url = (f'https://data.qiime2.org/{qiime2.__release__}/'
+                  'tutorials/metadata/rep-seqs.qza')
+rep_seqs_2_url = (f'https://data.qiime2.org/{qiime2.__release__}/'
+                  'tutorials/phylogeny/rep-seqs.qza')
+taxonomy_1_url = ('https://docs.qiime2.org/jupyterbooks/cancer-microbiome-'
+                  'intervention-tutorial/data/030-tutorial-downstream/020-'
+                  'taxonomy/taxonomy.qza')
+moving_pics_ft_url = (f'https://data.qiime2.org/{qiime2.__release__}/'
+                      'tutorials/filtering/table.qza')
+moving_pics_md_url = (f'https://data.qiime2.org/{qiime2.__release__}/'
+                      'tutorials/moving-pictures/sample_metadata.tsv')
+rep_seqs_dada2_url = 'https://docs.qiime2.org/2022.8/data/tutorials/' \
```

Comment:
Not sure if we should hard-code the QIIME 2 release in these URLs... I would try using the qiime2.__release__ for these rep seqs URLs and see if they work!

Reply:
Whoops yeah those are from an older commit and didn't get updated good catch